src/sfProject.ts

```diff
@@ -67,16 +35,10 @@ export type NamedPackageDir = PackageDir & {
   fullPath: string;
 };
 
-export type ProjectJson = ConfigContents & {
-  packageDirectories: PackageDir[];
-  namespace?: string;
-  sourceApiVersion?: string;
-  sfdcLoginUrl?: string;
-  signupTargetLoginUrl?: string;
-  oauthLocalPort?: number;
-  plugins?: { [k: string]: unknown };
-  packageAliases?: { [k: string]: string };
-};
+export type NamedPackagingDir = PackagePackageDir & NamedDirAdditions;
```

Comment:
What's the difference between "package" and "packaging" - would be nice to have comments here explaining

NIT: `NamedDirAdditions` also unreasonably iirks me, maybe just `NameAndPath` so that it becomes `PackagePackageDir & NameAndPath`?

Reply:
in sfdx-project.json they're called `packageDirectories` even if you're not using them for packaging.  `Packaging` refers to 1gp/2gp packages

I like you idea but will use `NameAndFullPath` since path is already on the basic one